frontend/packages/client/src/pages/Community.js

```diff
@@ -233,8 +230,8 @@ export default function Community() {
                   </span>
                 </p>
                 <div className="is-flex">
-                  {adminAddrs
-                    ? adminAddrs.slice(0, 5).map((adminAddr, idx) => (
+                  {admins
+                    ? admins.slice(0, 5).map((adminAddr, idx) => (
```

Comment:
```suggestion
  ? admins.slice(0, 5).map(({ addr: adminAddr }, idx) => {
```
admins is an array of objects that needs to use addr, otherwise the icons do not render correctly
<img width="184" alt="Screen Shot 2022-06-06 at 13 06 45" src="https://user-images.githubusercontent.com/7526740/172199933-f43f77dd-2ada-4ab1-baa0-4ba8910f6dea.png">